test/literal/result.test.rb

```diff
@@ -1,219 +1,3 @@
 # frozen_string_literal: true
 
-# # frozen_string_literal: true
```

Comment:
Need to replace these tests.

Reply:
Want a hand with them?

Reply:
I'm actually stalling because I'm working on GreenDots itself a bit first, trying to improve the failure output. 😅

Reply:
Hah ok!